README.md

```diff
@@ -128,6 +128,10 @@ In order to perform, a running instance of pulp with all necessary plugins insta
 All tests are excercised using `make test`.
 Individual tests can be selected by pattern via `pytest -k file`.
 
+### Pulp CLI feedback & thank you
+
+If you take the time to evaluate, test, or build your own CLI commands, please complete [this survey](https://forms.gle/ca1nxVVkNivEeE5m8), and we will gladly ship you some SWAG! 
+
```

Comment:
This note feels a little bit out of place between the other very technical ones. Maybe move it up as the first item in Contributing.